jekyll/_cci2/updating-server.adoc

```diff
@@ -76,6 +220,7 @@ Example Output:
 ```
 
 . For the replicated update to succeed, it is necessary to update docker to the recommended version, 17.12.1:
+// should there be a step to check Docker version first?
```

Comment:
ah that's probably a good thing to do before starting this whole process. They can do a version check by running `docker version` .

Reply:
Done